runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java

```diff
@@ -43,7 +43,12 @@
 import org.apache.beam.vendor.guava.v20_0.com.google.common.collect.ImmutableMap;
 import org.apache.beam.vendor.guava.v20_0.com.google.common.collect.Iterables;
 
-/** Cross-language external transform. */
+/**
+ * Cross-language external transform.
+ *
+ * <p>{@link External} provides a cross-language transform via expansion services in non-Java SDKs.
+ * This is a low-level API and mainly for internal use.
+ */
```

Comment:
Since this is part of the API now, we could expand the description here a bit, e.g. describing the requirements to use this transform (ExpansionService).

Reply:
Thanks. Added more comments on requirements.